src/Models/ContentCategoryHasContentsFilter.php

```diff
@@ -5,7 +5,7 @@
 class ContentCategoryHasContentsFilter extends Filter
 {
     public string $typeDefinition = "Relewise.Client.Requests.Filters.ContentCategoryHasContentsFilter, Relewise.Client";
-    public static function create(bool $negated = false) : ContentCategoryHasContentsFilter
+    public static function create(bool $negated) : ContentCategoryHasContentsFilter
```

Comment:
This is due to a breaking change in the .NET SDK. So we are okay with breaking the PHP SDK as well.